### ISSUE TITLE
Passing variables

### ISSUE THREAD

**Issue by esotm** [happycog/VMG-Nested#3]
Great add-on. Thank you. I see in the documentation that tag parameters can be passed. Is there a way to pass a variable (similar to how you can pass one with an embed) to use within the nested tag? 

I would like to be able to do something like this within a nested entry loop:
<pre><code>{exp:channel:entries channel="parts"}
  {parts-color}
  {exp:vmg_nested:channel:entries channel="colors" parts-color="{parts-color}"}
    {if "{embed:parts-color}" == "blue"}Blue{/if}
  {/exp:vmg_nested:channel:entries}
{/exp:channel:entries}
</pre></code>

**Comment by andrew-kaslick** [comment]
Hi @esotm,

Happy to hear you're finding the plugin useful!

Just to clarify, the issue you're having is with passing {parts-color} from the outer loop to a parameter of the nested loop? That should definitely be possible. Is the code above the actual code you're using in the template?

**Comment by esotm** [comment]
Yes the code is fairly similar. parts-color is just a custom field within the parts entry and I would like to pass it's data through to use within the nested tag.

**Comment by andrew-kaslick** [comment]
Sorry for the delay in responding. Since parts-color is a custom field, you'll need to use the [search: parameter](http://ellislab.com/expressionengine/user-guide/add-ons/channel/channel_entries.html#search-field-name) in the inner loop to filter entries by that value. Let me know if you still have issues passing that value as a parameter.
